libraries/helpers.rb

```diff
@@ -168,8 +187,13 @@ def platform_service_name(_version = node.run_state['mariadb']['version'])
       'mysql'
     end
 
-    def mysql_command_string(database, query)
-      "psql -d #{database} <<< '#{query};'"
+    def restart_mariadb_service
+      # Using this to generate a service resource to control
+      service 'mariadb' do
+        service_name platform_service_name
+        supports restart: true, status: true, reload: true
+        action :restart
+      end
```

Comment:
We'll probably need this resource later on in the run, so why are we not making a generic copy of it somwher? 🤔

Reply:
Because for now, we don't need it elsewhere 👿 ?